terraform-plans/templates/github/charm_promote.yaml.tftpl

```diff
@@ -8,7 +8,15 @@
 # https://github.com/canonical/charming-actions/issues/157
 # https://github.com/canonical/charmcraft/issues/2243
 
-# name: Promote charm to default track, standard risk levels.
+name: Promote charm to default track, standard risk levels.
```

Comment:
While we're here, can we rename this to something more concise please? This is very long, and is usually truncated in the UI.

```suggestion
name: Promote charm
```